src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,10 @@
+package org.example;
+
+public class Main {
+    public static void main(String[] args){
+
+        Second second = new Second();
+        int anInt = second.getInt();
+        System.out.println("~~~" + anInt + "~~~");
```

Comment:
This could be simplified to `System.out.println("~~~" + second.getInt() + "~~~");`, just suggestion